packages/react-tinacms-inline/src/inline-wysiwyg.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+import { InlineField } from './inline-field'
+import { Wysiwyg } from 'react-tinacms-editor'
+
+interface InlineWysiwygFieldProps {
+  name: string
+  sticky?: string
+  children: any
+}
```

Comment:
We should add [`format`](https://github.com/tinacms/tinacms/blob/fix-tina-provider-name/packages/react-tinacms-editor/src/Wysiwyg.tsx#L29-L34) to the props and pass it through.

```suggestion
interface InlineWysiwygFieldProps {
  name: string
  sticky?: string
  children: any
  format?: Format
}
```